code/iaas/model/src/main/java/io/cattle/platform/core/cleanup/TableCleanUp.java

```diff
@@ -0,0 +1,30 @@
+package io.cattle.platform.core.cleanup;
+
+import static io.cattle.platform.core.model.tables.ContainerEventTable.CONTAINER_EVENT;
+import static io.cattle.platform.core.model.tables.ProcessInstanceTable.PROCESS_INSTANCE;
+import static io.cattle.platform.core.model.tables.ServiceEventTable.SERVICE_EVENT;
+
+import io.cattle.platform.archaius.util.ArchaiusUtil;
+import io.cattle.platform.db.jooq.dao.impl.AbstractJooqDao;
+
+import java.util.Date;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import com.netflix.config.DynamicLongProperty;
+
+
+public class TableCleanUp extends AbstractJooqDao {
+
+    private static final Log logger = LogFactory.getLog(TableCleanUp.class);
+    private static final DynamicLongProperty CLEAN_UP_TIME = ArchaiusUtil.getLong("api.table.row.age");
```

Comment:
Change to something consistent with "task.purge.after.seconds",   I'd want to break this out into two settings.  process_instance.purge.after.seconds and events.purge.after.seconds